src/ONYXKEYS.ts

```diff
@@ -375,7 +376,8 @@ type OnyxValues = {
     // Collections
     [ONYXKEYS.COLLECTION.DOWNLOAD]: OnyxTypes.Download;
     [ONYXKEYS.COLLECTION.POLICY]: OnyxTypes.Policy;
-    [ONYXKEYS.COLLECTION.POLICY_CATEGORIES]: unknown;
+    [ONYXKEYS.COLLECTION.POLICY_CATEGORIES]: Record<string, OnyxTypes.PolicyCategory>;
+    [ONYXKEYS.COLLECTION.RECENTLY_USED_POLICY_CATEGORIES]: OnyxTypes.PolicyCategory[];
```

Comment:
And this one will be a **record of arrays of `PolicyCategory`**. Also expected?

Reply:
No, I need an array. Is there a way how to have a simple array?

Reply:
@rezkiy37 if you just need to store an array of items, the key must be outside of `COLLECTION`. e.g.

```ts
[ONYXKEYS.BETAS]: OnyxTypes.Beta[];
```

Reply:
Done - d59ddb1.